### DISCUSSION TITLE
Can I disable disk encryption?

### DISCUSSION THREAD

**Discussion by 0xMishell** [elementary/triage#544] [Q&A]
I have installed eOS Horus and disk encryption is enabled. The problem is I need to enter password on every start even if I just hybernate laptop for a while. Also I have a lot of configs, work documents and so on, so I dont want to reinstall whole system, just to remove encryption.

**Comment by teamcons** [comment]
Disk encryption works by encrypting everything. If you wish to disable it, it means you ask the system to decrypt everything, and replace everything with its decrypted version... It is possible, but risky, and i think you really would be better off copying all data and work documents somewhere else and reinstall

I also in the past regretted doing a full encryption for a computer where it wasnt relevant. But at least it's not the reverse, where you regret not encrypting it
